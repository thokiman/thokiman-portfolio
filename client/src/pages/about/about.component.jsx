import React from 'react';
import { connect } from 'react-redux';
import { Route, Switch, withRouter } from 'react-router-dom';
import { createStructuredSelector } from 'reselect';
import AboutEducation from '../../components/abouts/about-contents/about-content-education/about-content-education.component';
import AboutProject from '../../components/abouts/about-contents/about-content-project/about-content-project.component';
import AboutSkill from '../../components/abouts/about-contents/about-content-skill/about-content-skill.component';
import AboutSummary from '../../components/abouts/about-contents/about-content-summary/about-content-summary.component';
import AboutHomeContent from '../../components/abouts/about-contents/about-content/about-content.component';
import AboutDropdown from '../../components/abouts/about-dropdowns/about-dropdown/about-dropdown.component';
import AboutTimelineCareer from '../../components/abouts/about-timeline-careers/about-timeline-career/about-timeline-career.component';
import Footer from '../../components/footers/footer/footer.component';
import useWindowDimensions from '../../components/hooks/window-dimensions/useWindowDimensions.component';
import {
  selectAboutRoute,
  selectEducationRoute,
  selectProjectRoute,
  selectRockWaveImage,
  selectSkillEngineerRoute,
  selectSkillRoute,
  selectSkillTechnologyRoute,
  selectSkillVisualArtRoute,
  selectSummaryRoute,
  selectTimelineCareerRoute,
} from '../../redux/about/about.selectors';
import { selectIsSideBarActive } from '../../redux/header/header.selectors';
import { AboutElement, AboutIsNotActivated, AboutPage } from './about.styles';
import {
  measureAbouteHeight,
  measureAboutIsNotActivated,
} from './about.utils.styles';

export const About = ({
  location: { pathname },
  match,
  aboutRoute,
  skillRoute,
  aboutVisualArtSkillRoute,
  aboutEngineerSkillRoute,
  aboutTechnologySkillRoute,
  educationRoute,
  projectRoute,
  summaryRoute,
  timelineCareerRoute,
  isSideBarActive,
  rockWaveImage,
}) => {
  const { viewWidth, viewHeight } = useWindowDimensions();

  return (
    <>
      <AboutElement>
        <AboutPage
          $aboutheight={measureAbouteHeight(
            pathname,
            viewWidth,
            viewHeight,
            aboutRoute,
            skillRoute,
            aboutVisualArtSkillRoute,
            aboutEngineerSkillRoute,
            aboutTechnologySkillRoute,
            educationRoute,
            projectRoute,
            summaryRoute,
            timelineCareerRoute
          )}
          $matchpath={!!pathname.match(`${aboutRoute}${timelineCareerRoute}$`)}
          $issidebaractive={isSideBarActive}
          $rockWaveImage={rockWaveImage}
        >
          {isSideBarActive ? (
            <AboutIsNotActivated
              $aboutisnotactivatedprops={measureAboutIsNotActivated(
                viewWidth,
                viewHeight
              )}
            >
              Abo
              <br />
              ut
            </AboutIsNotActivated>
          ) : null}

          <AboutDropdown />
          <Switch>
            <Route exact path={aboutRoute} component={AboutHomeContent} />
            <Route path={skillRoute} component={AboutSkill} />
            <Route path={educationRoute} component={AboutEducation} />
            <Route path={projectRoute} component={AboutProject} />
            <Route path={summaryRoute} component={AboutSummary} />
            <Route
              path={`${aboutRoute}${timelineCareerRoute}`}
              component={AboutTimelineCareer}
            />
          </Switch>
        </AboutPage>

        <Footer />
      </AboutElement>
    </>
  );
};

const mapStateToProps = createStructuredSelector({
  aboutRoute: selectAboutRoute,
  skillRoute: selectSkillRoute,
  aboutVisualArtSkillRoute: selectSkillVisualArtRoute,
  aboutEngineerSkillRoute: selectSkillEngineerRoute,
  aboutTechnologySkillRoute: selectSkillTechnologyRoute,
  educationRoute: selectEducationRoute,
  projectRoute: selectProjectRoute,
  summaryRoute: selectSummaryRoute,
  timelineCareerRoute: selectTimelineCareerRoute,
  isSideBarActive: selectIsSideBarActive,
  rockWaveImage: selectRockWaveImage,
});

export default withRouter(connect(mapStateToProps)(About));
