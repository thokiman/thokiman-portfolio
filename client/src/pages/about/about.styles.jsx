import styled, { css, keyframes } from 'styled-components';

const whiteSmokeColor = '#e3e1e4';
const greenBlueColor = '#128194';

const backgroundColorTimelineCareerContainer = 'rgba(18, 129, 148, 0.8)';
const maxWidthSmartphone = 800;
const getAboutImage = ({ $rockWaveImage }) => {
  return $rockWaveImage.item.iconPath;
};

const aboutSideBarHide = keyframes`
  0% {
    transform: rotateX(100deg);
    transform-origin: bottom;
    opacity: 0;
  }
  100% {
    transform: rotateX(0);
    transform-origin: bottom;
    opacity: 1;
  }

`;

export const AboutIsNotActivated = styled.div`
  font-family: futura-demi;
  color: ${whiteSmokeColor};
  position: absolute;
  top: 19.5vh;
  left: 6vw;

  font-size: 10vw;
  letter-spacing: 0.17em;
  animation: ${aboutSideBarHide} 2s cubic-bezier(0.175, 0.885, 0.32, 1.275) both;
`;

const aboutHeight = ({ $aboutheight }) => {
  return $aboutheight;
};

const AboutIsActived = css`
  @media only screen and (max-width: ${maxWidthSmartphone}px) {
    position: relative;
    left: 0;
    width: 100vw;
    transition: left 0.5s ease;
    opacity: 1;
    flex-direction: column;
    justify-content: flex-start;

    height: ${aboutHeight}vh;
    overflow-y: hidden;
  }
`;
const getAboutSideBarActive = ({ $issidebaractive }) => {
  if ($issidebaractive) return null;
  return AboutIsActived;
};

const AboutStyles = css`
  opacity: 0.9;
  background-size: cover;
  background-repeat: no-repeat;
  display: flex;
  justify-content: space-between;
  align-items: center;
  font-family: futura-medium;
  transition: background-image 0.7s ease-in-out, height 0.4s ease-in-out;
  @media only screen and (max-width: ${maxWidthSmartphone}px) {
    position: relative;
    top: 0;
    width: 64vw;
    left: 140px;
    right: 0;
    bottom: 0;
    transition: left 0.5s ease;
    height: 115vh;
    overflow-y: scroll;
    overflow-x: hidden;
    opacity: 0.5;
    height: 70.8vh;
  }
  @media only screen and (max-width: 500px) {
    height: 68vh;
  }
  @media only screen and (max-width: 350px) {
    height: 71vh;
  }
  ${getAboutSideBarActive};
`;
const AboutContainer = css`
  ${AboutStyles}
  height: 72vh;

  background-image: url(${getAboutImage});
`;

const TimelineCareerContainer = css`
  ${AboutStyles}
  background-color: ${backgroundColorTimelineCareerContainer};
  padding-top: 40px;
  align-items: flex-start;

  @media only screen and (max-width: ${maxWidthSmartphone}px) {
    padding: 0;
    background-color: ${greenBlueColor};
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    align-items: center;
  }
`;

const getAboutPage = ({ $matchpath }) => {
  if ($matchpath) return TimelineCareerContainer;
  return AboutContainer;
};

export const AboutPage = styled.div`
  ${getAboutPage}
`;
AboutPage.displayName = 'AboutPage';
