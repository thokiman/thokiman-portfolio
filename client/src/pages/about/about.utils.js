import {
  hasScreenSize1,
  hasScreenSize2,
  hasScreenSize3_1,
  hasScreenSize3_2,
  hasScreenSize3_3,
  hasScreenSize3_4,
  hasScreenSize3_5,
  hasScreenSize3_6,
  hasScreenSize4,
  hasScreenSize5,
} from '../../utils/common.component.utils';

export const measureAbouteHeight = (
  pathname,
  viewWidth,
  viewHeight,
  aboutRoute,
  skillRoute,
  aboutVisualArtSkillRoute,
  aboutEngineerSkillRoute,
  aboutTechnologySkillRoute,
  educationRoute,
  projectRoute,
  summaryRoute,
  timelineCareerRoute
) => {
  let height;

  switch (pathname) {
    case aboutRoute:
      if (hasScreenSize1(viewWidth)) {
        height = 140;
      } else if (hasScreenSize2(viewWidth)) {
        height = 140;
      } else if (hasScreenSize3_1(viewWidth, viewHeight)) {
        height = 140;
      } else if (hasScreenSize3_2(viewWidth, viewHeight)) {
        height = 150;
      } else if (hasScreenSize3_3(viewWidth, viewHeight)) {
        height = 140;
      } else if (hasScreenSize3_4(viewWidth, viewHeight)) {
        height = 140;
      } else if (hasScreenSize4(viewWidth, viewHeight)) {
        height = 140;
      } else if (hasScreenSize5(viewWidth)) {
        height = 140;
      } else {
        height = 140;
      }
      return height;
    case skillRoute:
    case aboutVisualArtSkillRoute:
      if (hasScreenSize1(viewWidth)) {
        height = 300;
      } else if (hasScreenSize2(viewWidth)) {
        height = 330;
      } else if (hasScreenSize3_1(viewWidth, viewHeight)) {
        height = 300;
      } else if (hasScreenSize3_2(viewWidth, viewHeight)) {
        height = 270;
      } else if (hasScreenSize3_3(viewWidth, viewHeight)) {
        height = 320;
      } else if (hasScreenSize3_4(viewWidth, viewHeight)) {
        height = 350;
      } else if (hasScreenSize4(viewWidth, viewHeight)) {
        height = 400;
      } else if (hasScreenSize5(viewWidth)) {
        height = 320;
      } else {
        height = 330;
      }
      return height;
    case aboutEngineerSkillRoute:
      if (hasScreenSize1(viewWidth)) {
        height = 110;
      } else if (hasScreenSize2(viewWidth)) {
        height = 120;
      } else if (hasScreenSize3_1(viewWidth, viewHeight)) {
        height = 95;
      } else if (hasScreenSize3_2(viewWidth, viewHeight)) {
        height = 100;
      } else if (hasScreenSize3_3(viewWidth, viewHeight)) {
        height = 100;
      } else if (hasScreenSize3_4(viewWidth, viewHeight)) {
        height = 105;
      } else if (hasScreenSize4(viewWidth, viewHeight)) {
        height = 110;
      } else if (hasScreenSize5(viewWidth)) {
        height = 100;
      } else {
        height = 105;
      }
      return height;
    case aboutTechnologySkillRoute:
      if (hasScreenSize1(viewWidth)) {
        height = 485;
      } else if (hasScreenSize2(viewWidth)) {
        height = 600;
      } else if (hasScreenSize3_1(viewWidth, viewHeight)) {
        height = 555;
      } else if (hasScreenSize3_2(viewWidth, viewHeight)) {
        height = 565;
      } else if (hasScreenSize3_3(viewWidth, viewHeight)) {
        height = 595;
      } else if (hasScreenSize3_4(viewWidth, viewHeight)) {
        height = 650;
      } else if (hasScreenSize4(viewWidth, viewHeight)) {
        height = 710;
      } else if (hasScreenSize5(viewWidth)) {
        height = 565;
      } else {
        height = 607.5;
      }
      return height;
    case educationRoute:
      if (hasScreenSize1(viewWidth)) {
        height = 1210;
      } else if (hasScreenSize2(viewWidth)) {
        height = 1260;
      } else if (hasScreenSize3_1(viewWidth, viewHeight)) {
        height = 880;
      } else if (hasScreenSize3_2(viewWidth, viewHeight)) {
        height = 800;
      } else if (hasScreenSize3_3(viewWidth, viewHeight)) {
        height = 985;
      } else if (hasScreenSize3_4(viewWidth, viewHeight)) {
        height = 1000;
      } else if (hasScreenSize4(viewWidth, viewHeight)) {
        height = 1030;
      } else if (hasScreenSize5(viewWidth)) {
        height = 810;
      } else {
        height = 996.9;
      }
      return height;
    case projectRoute:
      if (hasScreenSize1(viewWidth)) {
        height = 1320;
      } else if (hasScreenSize2(viewWidth)) {
        height = 1380;
      } else if (hasScreenSize3_1(viewWidth, viewHeight)) {
        height = 970;
      } else if (hasScreenSize3_2(viewWidth, viewHeight)) {
        height = 920;
      } else if (hasScreenSize3_3(viewWidth, viewHeight)) {
        height = 1085;
      } else if (hasScreenSize3_4(viewWidth, viewHeight)) {
        height = 1110;
      } else if (hasScreenSize4(viewWidth, viewHeight)) {
        height = 1140;
      } else if (hasScreenSize5(viewWidth)) {
        height = 910;
      } else {
        height = 1107.5;
      }
      return height;
    case summaryRoute:
      if (hasScreenSize1(viewWidth)) {
        height = 300;
      } else if (hasScreenSize2(viewWidth)) {
        height = 305;
      } else if (hasScreenSize3_1(viewWidth, viewHeight)) {
        height = 230;
      } else if (hasScreenSize3_2(viewWidth, viewHeight)) {
        height = 215;
      } else if (hasScreenSize3_3(viewWidth, viewHeight)) {
        height = 250;
      } else if (hasScreenSize3_4(viewWidth, viewHeight)) {
        height = 245;
      } else if (hasScreenSize4(viewWidth, viewHeight)) {
        height = 245;
      } else if (hasScreenSize5(viewWidth)) {
        height = 205;
      } else {
        height = 249.4;
      }
      return height;
    case aboutRoute.concat('', timelineCareerRoute):
      if (hasScreenSize1(viewWidth)) {
        height = 290;
      } else if (hasScreenSize2(viewWidth)) {
        height = 400;
      } else if (hasScreenSize3_1(viewWidth, viewHeight)) {
        height = 390;
      } else if (hasScreenSize3_5(viewWidth, viewHeight)) {
        height = 430;
      } else if (hasScreenSize3_6(viewWidth, viewHeight)) {
        height = 505;
      } else if (hasScreenSize4(viewWidth, viewHeight)) {
        height = 600;
      } else if (hasScreenSize5(viewWidth)) {
        height = 530;
      } else {
        height = 458.3;
      }

      return height;
    default:
      return 800;
  }
};
