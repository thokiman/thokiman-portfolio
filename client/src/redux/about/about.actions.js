import AboutActionTypes from './about.types';

export const toggleVibrate = () => ({
  type: AboutActionTypes.SUMMARY_TOGGLE_VIBRATE,
});

export const toggleDownload = () => ({
  type: AboutActionTypes.SUMMARY_TOGGLE_DOWNLOAD,
});

export const toggleDropdownHidden = () => ({
  type: AboutActionTypes.PAGE_DROPDOWN_HIDDEN,
});

export const toggleYear2016Click = () => ({
  type: AboutActionTypes.TIMELINE_CAREER_YEAR_2016_CLICK,
});
export const toggleYear2017Click = () => ({
  type: AboutActionTypes.TIMELINE_CAREER_YEAR_2017_CLICK,
});
export const toggleYear2018Click = () => ({
  type: AboutActionTypes.TIMELINE_CAREER_YEAR_2018_CLICK,
});
export const toggleYear2019Click = () => ({
  type: AboutActionTypes.TIMELINE_CAREER_YEAR_2019_CLICK,
});
export const toggleYearCurrentClick = () => ({
  type: AboutActionTypes.TIMELINE_CAREER_YEAR_CURRENT_CLICK,
});
