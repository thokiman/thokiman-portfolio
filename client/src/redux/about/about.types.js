const AboutActionTypes = {
  SUMMARY_TOGGLE_VIBRATE: "SUMMARY_TOGGLE_VIBRATE",
  SUMMARY_TOGGLE_DOWNLOAD: "SUMMARY_TOGGLE_DOWNLOAD",
  PAGE_DROPDOWN_HIDDEN: "PAGE_DROPDOWN_HIDDEN",
  TIMELINE_CAREER_YEAR_2016_CLICK: "TIMELINE_CAREER_YEAR_2016_CLICK ",
  TIMELINE_CAREER_YEAR_2017_CLICK: "TIMELINE_CAREER_YEAR_2017_CLICK ",
  TIMELINE_CAREER_YEAR_2018_CLICK: "TIMELINE_CAREER_YEAR_2018_CLICK ",
  TIMELINE_CAREER_YEAR_2019_CLICK: "TIMELINE_CAREER_YEAR_2019_CLICK ",
  TIMELINE_CAREER_YEAR_CURRENT_CLICK: "TIMELINE_CAREER_YEAR_CURRENT_CLICK ",
};

export default AboutActionTypes;
