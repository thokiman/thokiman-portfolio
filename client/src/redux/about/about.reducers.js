import { about } from '../../data.object/about.data';
import { other } from '../../data.object/other.data';
import AboutActionTypes from './about.types';
const INTIAL_STATE = {
  about,
  other,
  isDropdownHidden: true,
  isVibrate: false,
  isDownloadClick: false,
  isYear2016Click: false,
  isYear2017Click: false,
  isYear2018Click: false,
  isYear2019Click: false,
  isYearCurrentClick: true,
  visualArtSkillImages: [],
  isLoadingVisualArtSkillImages: true,
};

const aboutReducer = (state = INTIAL_STATE, action) => {
  switch (action.type) {
    case AboutActionTypes.SUMMARY_TOGGLE_VIBRATE:
      return {
        ...state,
        isVibrate: !state.isVibrate,
      };
    case AboutActionTypes.SUMMARY_TOGGLE_DOWNLOAD:
      return {
        ...state,
        isDownloadClick: !state.isDownloadClick,
      };
    case AboutActionTypes.PAGE_DROPDOWN_HIDDEN:
      return {
        ...state,
        isDropdownHidden: !state.isDropdownHidden,
      };
    case AboutActionTypes.TIMELINE_CAREER_YEAR_2016_CLICK:
      return {
        ...state,
        isYear2016Click: !state.isYear2016Click,
        isYear2017Click: false,
        isYear2018Click: false,
        isYear2019Click: false,
        isYearCurrentClick: false,
      };
    case AboutActionTypes.TIMELINE_CAREER_YEAR_2017_CLICK:
      return {
        ...state,
        isYear2017Click: !state.isYear2017Click,
        isYear2016Click: false,
        isYear2018Click: false,
        isYear2019Click: false,
        isYearCurrentClick: false,
      };
    case AboutActionTypes.TIMELINE_CAREER_YEAR_2018_CLICK:
      return {
        ...state,
        isYear2018Click: !state.isYear2018Click,
        isYear2016Click: false,
        isYear2017Click: false,
        isYear2019Click: false,
        isYearCurrentClick: false,
      };
    case AboutActionTypes.TIMELINE_CAREER_YEAR_2019_CLICK:
      return {
        ...state,
        isYear2019Click: !state.isYear2019Click,
        isYear2016Click: false,
        isYear2017Click: false,
        isYear2018Click: false,
        isYearCurrentClick: false,
      };
    case AboutActionTypes.TIMELINE_CAREER_YEAR_CURRENT_CLICK:
      return {
        ...state,
        isYearCurrentClick: !state.isYearCurrentClick,
        isYear2016Click: false,
        isYear2017Click: false,
        isYear2018Click: false,
        isYear2019Click: false,
      };

    default:
      return state;
  }
};

export default aboutReducer;
