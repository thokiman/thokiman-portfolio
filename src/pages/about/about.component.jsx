import React from "react";
import { Route, Switch } from "react-router-dom";
import { connect } from "react-redux";
import { createStructuredSelector } from "reselect";

import PageShell from "../../components/page-shell/page-shell.component";
import AboutDropdown from "../../components/about-dropdown/about-dropdown.component";
import AboutHomeContent from "../../components/about-content/about-content.component";
import AboutSkill from "../../components/about-content-skill/about-content-skill.component";
import AboutEducation from "../../components/about-content-education/about-content-education.component";
import AboutProject from "../../components/about-content-project/about-content-project.component";
import AboutSummary from "../../components/about-content-summary/about-content-summary.component";
import AboutTimelineCareer from "../../components/about-timeline-career/about-timeline-career.component";

import { AboutPage } from "./about.styles";
import {
  selectAboutRoute,
  selectEducationRoute,
  selectProjectRoute,
  selectSkillRoute,
  selectSummaryRoute,
  selectTimelineCareerRoute,
} from "../../redux/about/about.selectors";
import { selectIsSideBarHidden } from "../../redux/header/header.selectors";

const About = ({
  location: { pathname },
  aboutRoute,
  skillRoute,
  educationRoute,
  projectRoute,
  summaryRoute,
  timelineCareerRoute,
  isSideBarHidden,
}) => {
  return (
    <AboutPage
      matchpath={!!pathname.match(`${aboutRoute}${timelineCareerRoute}$`)}
      $issidebarhidden={isSideBarHidden}
    >
      <AboutDropdown />
      <Switch>
        <Route exact path={aboutRoute} component={AboutHomeContent} />
        <Route path={skillRoute} component={AboutSkill} />
        <Route path={educationRoute} component={AboutEducation} />
        <Route path={projectRoute} component={AboutProject} />
        <Route path={summaryRoute} component={AboutSummary} />
        <Route
          path={`${aboutRoute}${timelineCareerRoute}`}
          component={AboutTimelineCareer}
        />
      </Switch>
    </AboutPage>
  );
};

const mapStateToProps = createStructuredSelector({
  aboutRoute: selectAboutRoute,
  skillRoute: selectSkillRoute,
  educationRoute: selectEducationRoute,
  projectRoute: selectProjectRoute,
  summaryRoute: selectSummaryRoute,
  timelineCareerRoute: selectTimelineCareerRoute,
  isSideBarHidden: selectIsSideBarHidden,
});

export default connect(mapStateToProps)(PageShell(About));
