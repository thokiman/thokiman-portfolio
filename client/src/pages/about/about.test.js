import React from "react";
import { shallow } from "enzyme";

import { About } from "./about.component";

describe("About page", () => {
  let wrapper;
  const mockRoute = "/";
  const mockFalseBool = false;
  beforeEach(() => {
    const mockProps = {
      location: { pathname: mockRoute },
      aboutRoute: mockRoute,
      skillRoute: mockRoute,
      educationRoute: mockRoute,
      projectRoute: mockRoute,
      summaryRoute: mockRoute,
      timelineCareerRoute: mockRoute,
      isSideBarHidden: mockFalseBool,
    };
    wrapper = shallow(<About {...mockProps} />);
  });
  it("should render About page", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should not render AboutSideBarHide if isSideBarHidden is false", () => {
    const mockProps = {
      location: { pathname: mockRoute },
      aboutRoute: mockRoute,
      skillRoute: mockRoute,
      educationRoute: mockRoute,
      projectRoute: mockRoute,
      summaryRoute: mockRoute,
      timelineCareerRoute: mockRoute,
      isSideBarHidden: mockFalseBool,
    };
    const newWrapper = shallow(<About {...mockProps} />);
    expect(newWrapper.exists("AboutSideBarHide")).toBe(false);
  });
});
