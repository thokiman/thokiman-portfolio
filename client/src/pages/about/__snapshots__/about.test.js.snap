// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page should render About page 1`] = `ShallowWrapper {}`;
